analyzer/src/main/kotlin/managers/Bundler.kt

```diff
@@ -35,4 +71,192 @@ class Bundler : PackageManager() {
+                           packages: MutableSet<Package>, errors: MutableList<String>) {
+        log.debug("Parsing scope: $groupName\nscope top level deps list=$dependencyList")
+        val scopeDependencies = mutableSetOf<PackageReference>()
+        dependencyList.forEach {
+            parseDependency(workingDir, it, packages, scopeDependencies, errors)
+        }
+        scopes.add(Scope(groupName, true, scopeDependencies.toSortedSet()))
+    }
+
+    private fun parseDependency(workingDir: File, gemName: String, packages: MutableSet<Package>,
+                                scopeDependencies: MutableSet<PackageReference>, errors: MutableList<String>) {
+        log.debug("Parsing dependency: $gemName")
+        try {
+
+            val (_, version, homepageUrl, declaredLicenses, description, dependencies) = getGemDetails(gemName,
+                    workingDir)
+            packages.add(Package(
+                    id = Identifier(
+                            provider = javaClass.simpleName,
+                            namespace = "",
+                            name = gemName,
+                            version = version
+                    ),
+                    declaredLicenses = declaredLicenses,
+                    description = description,
+                    homepageUrl = homepageUrl,
+                    binaryArtifact = RemoteArtifact.EMPTY,
+                    sourceArtifact = RemoteArtifact.EMPTY,
+                    vcs = parseVcs(homepageUrl))
+            )
+            val nonDevelDeps = dependencies.filter {
+                it["type"].asText() != ":development"
+            }
+            val transitiveDependencies = mutableSetOf<PackageReference>()
+            nonDevelDeps.forEach {
+                parseDependency(workingDir, it["name"].asText(), packages, transitiveDependencies, errors)
+            }
+            scopeDependencies.add(PackageReference(
+                    id = Identifier(
+                            provider = javaClass.simpleName,
+                            namespace = "",
+                            name = gemName,
+                            version = version
+                    ),
+                    dependencies = transitiveDependencies.toSortedSet())
+            )
+        } catch (e: Exception) {
+            if (com.here.ort.utils.printStackTrace) {
+                e.printStackTrace()
+            }
+
+            val errorMsg = "Failed to parse package (gem) $gemName: ${e.message}"
+            log.error { errorMsg }
+            errors.add(errorMsg)
+        }
+    }
+
+    // Gems tend to have github url set as homepage. Seems like it's the only way to get any vcs information.
+    private fun parseVcs(homepageUrl: String): VcsInfo =
+            if (Regex("https*:\\/\\/github.com\\/(?<owner>[\\w-]+)\\/(?<repo>[\\w-]+)").matches(homepageUrl)) {
```

Comment:
Maybe extract the `Regex` to a constant so it is not compiled on each function call?